packages/circuit-ui/components/BaseStyles/BaseStylesService.ts

```diff
@@ -43,6 +43,212 @@ export const createBaseStyles = ({
       url('${FONTS_BASE_URL}/aktiv-grotest-700.eot') format('embedded-opentype');
   }
 
+  :root {
+    // Normal backgrounds
```

Comment:
CSS comments should use a different syntax:

```suggestion
    /* Normal backgrounds */
```

Reply:
🙏 Yes. I'll remove these anyways before merging (I think the token names are pretty self-explanatory)

Reply:
I think they’re nice for grouping :)